src/components/FilterPanel.tsx

```diff
@@ -40,32 +40,19 @@ export default function FilterPanel({
   const [dialogOpen, setDialogOpen] = useState(false);
   const [params, setQueryParams] = useQueryParams();
 
-  const [minAmount, setMinAmount] = useState(params.minAmount);
-  const [grades, setGrades] = useState(params.grades);
-  const [majors, setMajors] = useState(params.majors);
-  const [states, setStates] = useState(params.states);
-  const location = useLocation();
-
-  useEffect(() => {
-    const query = queryString.parse(location.search, {
-      arrayFormat: 'bracket-separator',
-      arrayFormatSeparator: ',',
-      parseNumbers: true,
-    });
-
-    setMinAmount(query.minAmount);
-    setGrades(query.grades);
-    setMajors(query.majors);
-    setStates(query.states);
-  }, [location]);
+  const [minAmount, setMinAmount] = useState(params.minAmount || 0);
```

Comment:
This is maybe the one that shouldn't be defaulted as there is a semantic difference between explicit `0` and `undefined` :/

Reply:
It wouldn't matter though because in the `minAmountFilter` we would do `minAmount ?? 0`